spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -23,6 +23,30 @@ import java.io.File
 import java.security.MessageDigest
 import java.util.EnumSet
 
+private fun List<String>.addUppercaseAndCapitalized() = flatMap { listOf(it, it.toUpperCase(), it.capitalize()) }
+
+/**
+ * A list globs that match default license file names.
+ */
+val LICENSE_FILENAMES = listOf(
+    "license*",
+    "licence*",
+    "*.license",
+    "unlicense",
+    "unlicence",
+    "copying*",
+    "copyright",
+    "patents"
+).addUppercaseAndCapitalized()
+
+/**
+ * A list of globs that match files that often define the root license of a project, but are no license files and are
+ * therefore not contained in [LICENSE_FILENAMES].
+ */
+val ROOT_LICENSE_FILENAMES = listOf(
+    "readme*"
+).addUppercaseAndCapitalized()
```

Comment:
Currently it is required because readme files shall never be included in notice files. But the "patents" argument is correct, maybe adapting the docs would help? I'll think about it on Monday.

Reply:
Ping, just as a reminder.